go.sum

```diff
@@ -1 +1,2 @@
+github.com/DataDog/websites-modules v0.0.0-20210608134641-202ae42200c1 h1:Nf3QlFHeSC7Ye+Tnwq9EN5tT8OjV1W6E+qvnHTNWuy4=
```

Comment:
I am not sure what this does 🤔

Reply:
I'm not sure either, it showed up in my branch and I'm not sure how to reverse this?

Reply:
Gonna chat with Websites about what this file is but confirming that me and Kari reverted my addition so that there is only one line in this file (not two).